pkg/services/folder/registry.go

```diff
@@ -6,8 +6,10 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 )
 
+const AlertRuleKind = "alertrule"
```

Comment:
I wonder if it makes sense to move it to `/store/entity/models.go` so it will be in the same place as other kinds.